functional/systemd_test.go

```diff
@@ -35,7 +35,7 @@ func TestSystemdUnitFlow(t *testing.T) {
 	}
 	defer os.RemoveAll(uDir)
 
-	mgr, err := systemd.NewSystemdUnitManager(uDir)
+	mgr, err := systemd.NewSystemdUnitManager(uDir, false)
```

Comment:
@dongsupark Fixed `functional/systemd_test.go` here.
Thanks again for the reminder!